adam_core/utils/tests/test_benchmarks.py

```diff
@@ -1,52 +0,0 @@
-import pytest
-
-from .test_indexable import TestIndexable, concatenate
-
-
-@pytest.mark.parametrize(
```

Comment:
It might be nice to benchmark some of the new stuff. Shouldn't be in this revision though.